app/views/unattended/provisioning_templates/finish/kickstart_default_finish.erb

```diff
@@ -61,6 +63,8 @@ fi
 
 <%= snippet "blacklist_kernel_modules" %>
 
+<%= snippet_if_exists(template_name + " custom post") %>
```

Comment:
To prevent another extra empty like (we do not like them):

```suggestion
<%= snippet_if_exists(template_name + " custom post") -%>
```

The same above.

Reply:
Done, @lzap